src/sentry/south_migrations/0513_auto__del_grouptagvalue__del_unique_grouptagvalue_group_id_key_value__.py

```diff
@@ -0,0 +1,1577 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        # Removing unique constraint on 'EventTag', fields ['event_id', 'key_id', 'value_id']
+        db.delete_unique(u'sentry_eventtag', ['event_id', 'key_id', 'value_id'])
+
+        # Removing unique constraint on 'TagValue', fields ['project_id', 'key', 'value']
+        db.delete_unique('sentry_filtervalue', ['project_id', 'key', 'value'])
+
+        # Removing unique constraint on 'GroupTagKey', fields ['project_id', 'group_id', 'key']
+        db.delete_unique(u'sentry_grouptagkey', ['project_id', 'group_id', 'key'])
+
+        # Removing unique constraint on 'TagKey', fields ['project_id', 'key']
+        db.delete_unique('sentry_filterkey', ['project_id', 'key'])
+
+        # Removing unique constraint on 'GroupTagValue', fields ['group_id', 'key', 'value']
+        db.delete_unique('sentry_messagefiltervalue', ['group_id', 'key', 'value'])
+
+        # Deleting model 'GroupTagValue'
+        db.delete_table('sentry_messagefiltervalue')
+
+        # Removing index on 'GroupTagValue', fields ['project_id', 'key', 'value', 'last_seen']
+        db.delete_index('sentry_messagefiltervalue', ['project_id', 'key', 'value', 'last_seen'])
+
+        # Deleting model 'TagKey'
+        db.delete_table('sentry_filterkey')
+
+        # Deleting model 'GroupTagKey'
+        db.delete_table(u'sentry_grouptagkey')
+
+        # Deleting model 'TagValue'
+        db.delete_table('sentry_filtervalue')
+
+        # Removing index on 'TagValue', fields ['project_id', 'key', 'last_seen']
+        db.delete_index('sentry_filtervalue', ['project_id', 'key', 'last_seen'])
+
+        # Deleting model 'EventTag'
+        db.delete_table(u'sentry_eventtag')
```

Comment:
I don't think you can delete the table without removing the models from the source, otherwise the tables will be recreated the first time someone generates a migration.

Reply:
Yes the ORM models need to be removed before the tables are dropped.

Reply:
Deleted the models as well. Although I think it wouldn't find them since tagstore is gone from INSTALLED_APPS.